### DISCUSSION TITLE
[Kavita] Separate series entry per language of a series

### DISCUSSION THREAD

**Discussion by pe1uca** [Kareadita/Kavita#3305] [Ideas]
### Idea Description

I'd like to see a new entry for another language of the same series.  
Something like:

- Series name
- Series name (es)
- Series name (fr)
- Series name (jp)

---  

Currently kavita also has an issue with using the same series name for different languages.  

I have the same series in different directories (e.g."Series name" and "Series name French")  
The ComicInfo tag `title` of both has the same name, the `LanguageISO` tag is set accordingly.  
When adding them to the library it shows up only with one set of files under the series' name.  
When the task "Scan Library" is ran the series changes to the set of files of the other language and keeps alternating each time the task runs.  

### Idea Category

Web UI

### Duration of Using Kavita

2 years

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by scare376** [comment]
Most users that have series in multiple languages won't have them all in the same library. I for example have different libraries because it helps keep things cleaner for my users who dont need access to say my japanese library.

**Reply by pe1uca** [reply]
Oh, I see what you mean, I just moved one to another directory and created a new library. It now properly shows them separated.  

I have some small series in single directory, which has a few hundred files so it'll be annoying to sort out hehe.  
They are automatically sorted by kavita with the xml file into the proper series as opposed to parsing the file names and paths.  

Probably there's still value in sorting out the issue with only showing a set of files if they are in the same library, since randomly showing up a set of files is very confusing.
